src/types/onyx/PurchaseList.ts

```diff
@@ -0,0 +1,188 @@
+/** Subscription type for a purchase */
+type Subscription = {
+    /** Whether new users are added automatically */
+    addNewUsersAutomatically?: boolean;
+
+    /** Whether the subscription auto-renews */
+    autoRenew?: boolean;
+
+    /** Date when auto-renew was last changed */
+    autoRenewLastChangedDate?: string;
+
+    /** End date of the subscription */
+    endDate?: string;
+
+    /** Start date of the subscription */
+    startDate?: string;
+
+    /** Type of subscription */
+    type?: string;
+
+    /** Number of users in the subscription */
+    userCount?: number;
+};
+
+/** Type for a billable policy */
+type BillablePolicy = {
+    /** List of actors in the policy */
+    actorList?: string;
+    /** Approved spend amounts by currency */
+    approvedSpend?: Record<string, number>;
+    /** Whether the policy is corporate */
+    corporate?: boolean;
+    /** Expensify card spend by currency */
+    expensifyCardSpend?: Record<string, number>;
+    /** Type of the policy */
+    type?: string;
```

Comment:
NAB but should we put blank lines in between for consistency with the rest of the file?